documentation/autoinstall-reference.md

```diff
@@ -311,6 +311,24 @@ When using the "lvm" layout, LUKS encryption can be enabled by supplying a passw
 
 The default is to use the lvm layout.
 
+#### sizing-policy
+
+The lvm layout will, by default, attempt to leave room for snapshots and further expansion.  A sizing-policy key may be supplied to control this behavior.
+
+**type:** string (enumeration)
+**default:** scaled
+
+Supported values are:
+
+ * `scaled` -> adjust space allocated to the root LV based on space available to the VG
+ * `all` -> allocate all remaining VG space to the root LV
+
+The scaling system is currently as follows:
+ * Less than 10 GB: use all remaining space for root filesystem
```

Comment:
If applicable, I would prefer to use the `GiB` notation ; although I understand some people are uncomfortable with it. Being the reference document, I think it should reflect the actual implementation.

Reply:
This makes sense.  Subiquity needs to be specific in actual sizing.